packages/rfw/test/runtime_test.dart

```diff
@@ -347,6 +347,17 @@ void main() {
     expect(tester.widget<ColoredBox>(find.byType(ColoredBox)).color, const Color(0x00000002));
   });
 
+  testWidgets('Runtime', (WidgetTester tester) async {
+    final Runtime runtime = Runtime()
+          ..update(const LibraryName(<String>['core']), createCoreWidgets());
+    for (final MapEntry(key: LibraryName libName, value: WidgetLibrary lib) in runtime.libraries.entries) {
+      expect(libName.toString(), equals('core'));
+      if (lib case final LocalWidgetLibrary localWidgetLibrary) {
+        expect(localWidgetLibrary.widgets.length, greaterThan(1));
```

Comment:
why is this expectation conditional?

Reply:
In this test it would have always executed but I've rewritten this section to not use any conditional logic.